bims/static/js/views/olmap_layers.js

```diff
@@ -11,13 +11,6 @@ define(['shared', 'backbone', 'underscore', 'jquery', 'ol', 'views/layer_style']
         initialize: function () {
             this.layerStyle = new LayerStyle();
         },
-        isBiodiversityLayerShow: function () {
```

Comment:
is it ok to be removes?

Reply:
yes, there is another function that exactly the same (same name, same body). I just remove one of them.